pennylane_lightning/lightning_gpu/_mpi_handler.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2022-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the :class:`~.LightningGPU_MPIHandler` class, a MPI handler to use LightningGPU device with multi-GPU on multi-node system.
+"""
+
+try:
+    # pylint: disable=no-name-in-module
+    from pennylane_lightning.lightning_gpu_ops import DevTag, MPIManager
+
+    MPI_SUPPORT = True
+except ImportError:
+    MPI_SUPPORT = False
+
+from typing import Callable, Union
+
+import numpy as np
+
+
+# MPI options
+class LightningGPU_MPIHandler:
+    """MPI handler for PennyLane Lightning GPU device
+
+    MPI handler to use a GPU-backed Lightning device using NVIDIA cuQuantum SDK with parallel capabilities.
+
+    Use the MPI library is necessary to initialize different variables and methods to handle the data across nodes and perform checks for memory allocation on each device.
+
+    Args:
+        mpi (bool): declare if the device will use the MPI support.
+        mpi_buf_size (int): size of GPU memory (in MiB) set for MPI operation and its default value is 64 MiB.
+        dev_pool (Callable): Method to handle the GPU devices available.
+        num_wires (int): the number of wires to initialize the device with.
+        c_dtype (np.complex64, np.complex128): Datatypes for statevector representation
```

Comment:
End with a dot to keep style uniform.
```suggestion
        c_dtype (np.complex64, np.complex128): Datatypes for statevector representation.
```